vowpalwabbitslim/ut_explore.cpp

```diff
@@ -0,0 +1,258 @@
+#include "gtest/gtest.h"
```

Comment:
Minor:  consider a separate folder for tests

Reply:
I didn't do so to avoid having yet another Makefile. @jl@hunch.net  any preference?

---
In reply to: [188318281](https://github.com/JohnLangford/vowpal_wabbit/pull/1481#discussion_r188318281) [](ancestors = 188318281)